ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java

```diff
@@ -324,17 +323,23 @@ public void navigateTo(String title) {
 
     @When("^.*navigates? to the \"([^\"]*)\" page in help menu$")
     public void navigateToHelp(String title) {
-        SelenideElement helpDropdownMenu = $(By.className("help")).shouldBe(visible);
+        SelenideElement helpDropdownMenu = $(By.id("helpDropdownButton")).shouldBe(visible);
 
-        if (!helpDropdownMenu.getAttribute("class").contains("open")) {
+        //open the help menu
+        if (helpDropdownMenu.parent().$$(By.className("pf-c-dropdown__menu")).size() < 1) {
             helpDropdownMenu.click();
         }
 
-        SelenideElement dropdownElementsTable = $(By.className("dropdown-menu")).shouldBe(visible);
+        SelenideElement dropdownElementsTable = $(By.className("pf-c-dropdown__menu")).shouldBe(visible);
         ElementsCollection dropdownElements = dropdownElementsTable.findAll(By.tagName("a"))
                 .shouldBe(CollectionCondition.sizeGreaterThanOrEqual(1));
 
         dropdownElements.filter(text(title)).shouldHaveSize(1).get(0).shouldBe(visible).click();
+
+        //close the help menu
+        if (helpDropdownMenu.parent().$$(By.className("pf-c-dropdown__menu")).size() >= 1) {
+            helpDropdownMenu.click();
+        }
```

Comment:
maybe add a todo here to remove it when the issue is fixed?

Reply:
Well, it wont do anything after it gets fixed so it doesn't matter... But sure, why not.